tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -962,6 +962,12 @@ vlan/test_vlan_ping.py:
 #######################################
 #####             voq             #####
 #######################################
+voq:
+  skip:
+    reason: cisco-8000 platform is not VOQ based thus all VOQ testcases are not applicable to this platform and should be skipped
+    conditions:
+      - "asic_type in ['cisco-8000'] and len(VOQ_INBAND_INTERFACE) == 0"
```

Comment:
remove cisco check so that it will becomes generic. Does it work fine ?  It does not get over-written by more specific test case  below ?

Reply:
Done. And validated that It does not get over-written by more specific test case below.